collector/config.yaml

```diff
@@ -0,0 +1,17 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+
+exporters:
+  awsxray:
+  awsemf:
+
+service:
+  pipelines:
+    traces:
+      receivers: [otlp]
+      exporters: [awsxray]
```

Comment:
We need to come to a decision on whether it's ok for a repository in `open-telemetry` to host vendor-specific configs, distros, examples. I'm personally not ok with it and would recommend this be in an aws distro repo. Yes it's easier for a user to start if using the aws distros, but I would suggest that in that case this repo doesn't really belong here which has been my general stance. 

@open-telemetry/technical-committee can you advice on this, which originated from

https://github.com/open-telemetry/community/issues/591#issuecomment-800764602
https://github.com/open-telemetry/opentelemetry-lambda/pull/14#discussion_r595603941

There was some concern about TC oversight if in a different repo by @dyladan 

https://github.com/open-telemetry/community/issues/591#issuecomment-756534589

But this doesn't seem to be too big of a deal since instrumentation can still live in open-telemetry, and has already started with https://github.com/open-telemetry/opentelemetry-specification/pull/1442 (assuming it ever gets merged :P). I assume this is the focus of the TC, the instrumentation experience. The actual end user experience is more of a vendor concern IMO - in this case it would be AWS balancing its hat as a cloud vendor and an observability vendor, but possibly then optimizing for an AWS getting-started experience.